executable/SummarizeApp.re

```diff
@@ -32,60 +33,15 @@ let mockedNotification: User.Notification.t = {
   payload: "Payload",
 };
 
-let mockedMessages: list(User.Message.t) = [
+let mockedMessages: list(Message.t) = [
```

Comment:
these mocked data can be deleted, no? 🤔

Reply:
since we don't have database connection set I'm using this to test for now, gonna remove later